.gitignore

```diff
@@ -0,0 +1,5 @@
+./venv
+__pycache__
+Tracker.db
```

Comment:
Adding this to the git ignore so you can have a local db with what ever you want in and not push to gh :)